### ISSUE TITLE
License change plan 

### ISSUE THREAD

**Issue by cowboy93** [Patric-R/jvmtop#114]
Hi 
I want to distribute this jvmtop with source to everyone
But this jvmtop is GPL V2 licensed. 

Don't you have a plan to change license to Apache license ? 

Regards.

**Comment by kamstrup** [comment]
I what capacity does the GPL2 license prevent you from distributing? Binary distribution of GPL2 code is basically what all linux distros are based on so pretty sure it's a safe bet ;-) (as long as you make the source available via similar distribution means)

**Comment by cowboy93** [comment]
what capacity does the GPL2 license prevent you from distributing?
==>
No.
But my company's basic open source contributing rule is using Apache license. 
That's why I asked to him.

**Comment by kant111** [comment]
Yeah, why is it not Apache License? moreover, there is no release since 2015?  I think if the License is Apache many people would want to contribute and probably get more maintainers. This project is awesome, not sure why it is in a dormant state?
